starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,68 @@
+
+* {
+  margin: 0;
+  padding: 0;
+}
+body {
+  font-family: 'Open Sans', sans-serif;
+}
+nav {
+  background-image: url("../images/apple-pie.jpg");
+  background-size: cover;
+  height: 30rem;
```

Comment:
Cuidado con usar rems! Hay que tener un font-size de referencia antes de ponerlos, os paso un enlace para que podáis curiosear :) [https://www.w3schools.com/cssref/css_units.asp](url)